.github/workflows/nvda-test.yml

```diff
@@ -193,6 +201,7 @@ jobs:
 
       - name: upload *.{log,png}
         uses: actions/upload-artifact@v4
+        if: always()
```

Comment:
```suggestion
        if: ${{ always() }}
```

Reply:
I actually believe it's preferred to not use the expression syntax for always by itself. 

https://github.com/w3c/aria-at-app/pull/961/files#diff-fcc964e27d1e22339051a8ce39304eb2c721e3d96ab72d9fb20b4a2192f74b60R47

I seem to recall reading old issues saying that using the expression syntax with one of these alone was actually causing errors

Reply:
Actually - my memory was slightly incorrect, seems you can omit the expression syntax entirely.

https://stackoverflow.com/questions/58858429/how-to-run-a-github-actions-step-even-if-the-previous-step-fails-while-still-f#comment112735158_58859404

https://github.com/bocoup/aria-at-gh-actions-helper/pull/14/commits/1390571c3c91a9773fa4fb80ed62b5c50d69560c

https://github.com/bocoup/aria-at-gh-actions-helper/actions/runs/8472751048/job/23215516701 is a test run with no expression braces at all

Reply:
My preference is for the most consistent syntax even at the expense of verbosity for the same reason I don't lean on ASI when writing JS--honoring [the exceptions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif) results in more cognitive load and more risk than I think is warranted. I'm willing to be flexible on a few lines, though.